src/Codec/JVM/ASM/Code/Instr.hs

```diff
@@ -32,7 +32,8 @@ data InstrState =
              , isLabelTable    :: LabelTable
              , isLastBranch    :: LastBranch
              , isRunAgain      :: Bool
-             , isNextLabel     :: Int }
+             , isNextLabel     :: Int 
+             , isLineNumber    :: Maybe Int }
```

Comment:
No problem! I guess the naming is a bit off as well. `Instr` represents a monadic action, which when executed generates bytecode and other relevant information like attributes that can almost directly be written to classfiles.